app/controllers/application_controller.rb

```diff
@@ -38,6 +38,10 @@ def info_for_paper_trail
     { ip: request.env['HTTP_X_REAL_IP'] || request.remote_ip }
   end
 
+  def meta
```

Comment:
no need to add meta to the application controller. only for API specific controllers